setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
     marshmallow>=3.5.0,<4.0.0
     marshmallow-oneofschema>=2.1.0
     pycountry>=18.12.8
+    tinycss2>=1.2.1
```

Comment:
what is it used for?

Reply:
Actually should be specified as [a `bleach[css]` extra](https://github.com/mozilla/bleach/blob/5d9b65da839782ad348cec04bed2bf590c23353a/setup.py#L34). It's needed for the CSSSanitizer that goes through the `styles` attribute value for tables.